runtime/shared_common/OSCache.cpp

```diff
@@ -141,7 +153,7 @@ IDATA
 SH_OSCache::removeCacheVersionAndGen(char* buffer, UDATA bufferSize, UDATA versionLen, const char* cacheNameWithVGen)
 {
 	char* nameStart;
-	UDATA lengthMinusGen;
+	UDATA lengthMinusGenAndLayer;
```

Comment:
Since this is modified, please initialize it.

Reply:
Fixed